site/docs/guides/sprites.md

```diff
@@ -107,6 +120,13 @@ w4.blit(smiley.ptr, 10, 10, 8, 8, w4.blit1Bpp);
 w4.Blit(&smiley[0], 10, 10, 8, 8, w4.BLIT_1BPP)
 ```
 
+```odin
+// BLIT_1BPP is the default.
+// To use BLIT_2BPP:
+// w4.blit(&smiley[0], 10, 10, 8, 8, { .Use_2BPP })
+w4.blit(&smiley[0], 10, 10, 8, 8)
```

Comment:
These constants don't match the Odin file constants

Reply:
You're right. I had an earlier version which had the `Use_2BPP` instead of `USE_2BPP`.